### PR TITLE
feat: %f, %i replacements

### PR DESCRIPTION
**Pull request by commenthol**
Adds support for %i, %f replacements as of Issue #31

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -23,7 +23,6 @@ function format(f, args, opts) {
   }
   var argLen = args.length
   if (argLen === 0) return f
-  var x = ''
```
Comment (mcollina) [inline index.js:26]: why was this removed?

**Review by mcollina — COMMENTED** [review]
good work!

File: index.js (hunk 1 above)
Comment (commenthol) [reply index.js:26]: This variable is no longer used in the code. Seams that it has been used in previous versions.

**Review by mcollina — APPROVED** [review]
lgtm

**Review by davidmarkclements — APPROVED** [review]
nice job!
